packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -166,28 +167,20 @@ export class OfficeAddinUsageData {
    * Reports error to usage data structure
    * @param errorName Error name sent to usage data structure
    * @param err Error sent to usage data structure
+   * @deprecated Use reportException instead
    */
   public async reportError(errorName: string, err: Error): Promise<void> {
-    if (this.getUsageDataLevel() === UsageDataLevel.on) {
-      this.reportErrorApplicationInsights(errorName, err);
-    }
+    this.reportException(errorName, err);
   }
 
   /**
    * Reports error to Application Insights
    * @param errorName Error name sent to Application Insights
    * @param err Error sent to Application Insights
+   * @Deprecated Use reportException instead
```

Comment:
I looked at this more closely . . . they are not the same thing, though they are close.  This is lighter weight and a fall back for the reportException method and you are removing that safe guard.

Reply:
I will make it not public them, so it is only used in one place

Reply:
It might already be used in other places given that is has been public.  Before making a change to this at all we should look more carefully at why the two are being used and where they are being used outside of this project.

Reply:
I would also make a major update. I will keep it public to make it safe.

Reply:
You've still trying to deprecate it and use the other method.  I'm saying there shouldn't be any logging method changes in the PR that aren't specific to the new ID.

Reply:
Don't include this change in this PR